[release/1.1.0] Release 1.1.0

jesmrec opened the pull request:
<!--
Thanks for submitting a change to ownCloud!

To make it possible for us to get your change reviewed and merged please fill out below information carefully.

Please note that any kind of change first has to be submitted to the master branch which holds the next major version of ownCloud.
-->

## Description
<!--- Describe your changes in detail -->

## Related Issue
<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->
#461 

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

felix-schwarz committed:
Naming improvements based on latest SDK:
- uploads use new OCCoreOptionAutomaticConflictResolutionNameStyle option to automatically resolve naming conflicts during upload
- Duplicate action uses the new OCCore name suggestion API to
	- determine naming conflicts and resolve them automatically
	- match the name style of the file to duplicate, f.ex.
		- duplicating "File copy.jpg" will create "File copy 2.jpg"
		- duplicating "File (1).jpg" will create "File (2).jpg"
		- duplicating "File Kopie 2.jpg" will create "File Kopie 3.jpg"
- folder creation uses the new OCCore name suggestion API to
	- detect naming conflicts beforehand
	- pre-fill the new folder name name with an unused name directly

hosy committed:
#393 added an activity indicator which will be shown, if offline copies will be deleted to give the user a UI feedback

felix-schwarz committed:
Merge branch 'master' into feature/naming

felix-schwarz committed:
- Change SDK branch to master

hosy committed:
#76 user can import files using the iOS share sheet. all file types are accepted.

felix-schwarz committed:
Merge remote-tracking branch 'origin/master' into feature/naming

hosy committed:
- using new SDK with fixes for requesting core
- minor code fixes

felix-schwarz committed:
- Log device, version and locale information at the beginning of every log file using new SDK protocol
- Show SDK commit hash in Settings

hosy committed:
fixed crash, changed bookmark name to shortname

hosy committed:
changed UIAlertViewController for account selection to CardViewController style for testing

hosy committed:
fixed icon badge creation for fastlane lane In-House Enterprise IPA generation

hosy committed:
install librsvg for fastlane via sh

hosy committed:
moved import file code to own class

hosy committed:
added option to automatically resolve name conflicts, if file name already exists

hosy committed:
Changed app version to 1.1.0

hosy committed:
enabled beta build and warning

mneuwert committed:
Keep the gallery alive after doing some action over an item (#447)

* Rename updates the currently visible file in the gallery

* Keeping gallery alive fix

- Don’t pop to the root view controller on destructive action by default
- Only pop to the root if all items have been deleted
- Select next item upon deletion of the current ones
- Update UIPageViewController data source if number of items changes due e.g. to duplication or deletion

* Small fixes

* Fixed review findings

mneuwert committed:
Media player implemented using AVKit (#429)

* Media player implemented using AVKit

* Added possibility to stream audio / video

* Improved error handling

* Fixed review finding

* Added background audio, airplay and PiP mode

* Propagating safe area to the main view of AVPlayerViewController

* Another fix for safe area considering layout of AVPlayerViewController

* Updated to latest develop version of SDK

hosy committed:
Merge branch 'feature/log-info' into milestone/1.1
Log device, version and locale info #446

hosy committed:
Merge branch 'fix/delete-offline-activity-indicator' into milestone/1.1
Added an activity indicator for deleting offline copies #438

hosy committed:
Merge branch 'feature/naming' into milestone/1.1
Naming improvements for upload, duplicate and folder creation #430

mneuwert committed:
Display error message in case file couldn’t be preview e.g. due to file corruption (#427)

hosy committed:
Merge branch 'milestone/1.1' of github.com:owncloud/ios-app into milestone/1.1

hosy committed:
Fixed UI test for creating folder, need to return a fixed name, because suggestion is not working in tests

hosy committed:
added a description header and changed typo

hosy committed:
- fixed showing the directory picker controller, after the card view was dismissed
- use the current development SDK

Commit:
Merge branch 'milestone/1.1' into feature/share-sheet

Commit:
Version Bump to 126

hosy committed:
- create local copy of import file, if needed
- fixed create folder action
- delete local copy, if needed
- code review changes

felix-schwarz committed:
- improved duplicate item deletion behaviour
	- add additional safeguard so duplicate files are only deleted if they were actually duplicated (previously non-duplicate files could be removed if duplication or folder creation failed)
	- remove temporary container folder, too
- add 2 second delay before returning the core to give the core an opportunity to schedule the upload on a NSURLSession

hosy committed:
Merge branch 'feature/share-sheet' into milestone/1.1

Commit:
Version Bump to 127

hosy committed:
changed back signing identity

hosy committed:
Merge branch 'milestone/1.1' of github.com:owncloud/ios-app into milestone/1.1

Commit:
Version Bump to 128

hosy committed:
added required CFBundleTypeName key

hosy committed:
Merge branch 'milestone/1.1' of github.com:owncloud/ios-app into milestone/1.1

Commit:
Version Bump to 129

hosy committed:
updated changelog

hosy committed:
use formSheet presentation style for the iPad when showing the document picker

mneuwert committed:
Fix for images not being displayed in the gallery

Commit:
Version Bump to 130

Commit:
Version Bump to 131

Commit:
[fix/sharing-search] Fixed min length for searching sharing users (#455)

* #454 used correct comparator for sharingSearchMinLength and set a new default value, if capability does not exists

* added a constant for defaultSharingSearchMinLength value

felix-schwarz committed:
- Update ios-sdk to address finding (1) in ios-app #446

mneuwert committed:
Preventing updating UI in DisplayViewController while item is being changed

- Not calling present(item:) while meta data of OCItem is being updated
- Elliminated an assumption in MediaDisplayViewController that render renderSpecificView() can be called only once.
- Making sure that AVPlayer is not re-created upon changes in OCItem but that rather AVPlayerItem is replaced.

mneuwert committed:
Fixed a small warning

mneuwert committed:
Merge branch 'milestone/1.1' of https://github.com/owncloud/ios-app into milestone/1.1

mneuwert committed:
Tried to improve gallery logic concerning items modification

- Watch out if the modification does still includes the item with local ID of the currently visible item.
- Take care of failed move but watching out for reappearing items

mneuwert committed:
Added a setting allowing to decide user if media files shall be streamed

felix-schwarz committed:
- Add debug output to Display*ViewController
- Fix SwiftLint warnings

mneuwert committed:
Fixed handling of deleted / moved item in the gallery

mneuwert committed:
Merge branch 'milestone/1.1' of https://github.com/owncloud/ios-app into milestone/1.1

mneuwert committed:
Another small fix for handling of failed item move

Commit:
Version Bump to 132

mneuwert committed:
Added LSSupportsOpeningDocumentsInPlace key to Info.plist

mneuwert committed:
Merge branch 'milestone/1.1' of https://github.com/owncloud/ios-app into milestone/1.1

mneuwert committed:
Fixed Info.plist and added LSHandlerRank property

mneuwert committed:
Fix for #455 issue: no search request triggered

hosy committed:
show always the account selection sheet and added a note, that only one file can be imported at once

hosy committed:
use securtiy scoped file operation for importing a file

mneuwert committed:
Fix for the PR #447 (keep gallery alive) (#465)

* Keeping track of individual OCItems in DisplayViewController instances

But loading the list of items in the gallery only once and not reacting to any changes (moving, deleting)

* Removed query stop call in DisplayHostViewController

Since this query is not created there but just passed from the parent view.

* Removing more button if the currently viewed file got moved or deleted

* Fixed updating UI after renaming current item

* Fixed a warning

* Fixed issues in the gallery

* - Minor fixes

felix-schwarz committed:
[feature/bundle-import] Add support for certain bundle formats to the share sheet (#471)

* - Add support for import of bundle-based document formats through transparent zipping
	- moved previous FileProvider functionality into ownCloudApp.framework
	- new method OCCore.importItemNamed() ensures correct handling of bundle-based document formats
	- fixes finding (4) in #445
- Changed LSSupportsOpeningDocumentsInPlace to NO address two issues:
	- the app is now listed as "Copy to ownCloud" (previously "Open in ownCloud", which was technically not correct)
	- previously, documents using "Open to ownCloud" would no longer open in f.ex. Pages, but in the ownCloud app when selected in the Files app

* - Refactored ImportFilesController around using NSFileCoordinator, fixing finding (1) in #471

* changed path for moved Info.plist for target ownCloudAppFramework

Commit:
Version Bump to 133

Commit:
[feature/sort direction] Allow sorting asc/desc for all sort methods (#474)

* #470 added change sort direction asc/desc for all sort methods

* - removed comments
- changed translation keys

* fixes sort comparator for sort method "shared"

* use correct images for sort direction asc and desc

* updated choosing sort method from a tableView/Popover instead of a UIAlertViewController

* set maximum width for popover and set arrow direction

* Added show sort direction in UISegmentControl

* - make sure arrows of popovers appear in same color as the popover contents
- fix issue where direction was toggled during setup of SortBar (setting of initial value from user defaults), so logging in, logging out and logging in would change the sort direction
- reset sort direction to ascendant when switching sort methods
- SortMethodTableViewController
	- remove extraneous instance lets
	- enforce row height used for table view height calculation
	- change table view height calculation to account for the last divider (which could look a bit off if present)

* tap on segment control was fired twice, because there was a missing check, if selected segment changed

* fixed changing width on selecting an other segment

* fixed jumping width of sortSegmentControl items by setting the same width for all items by calculating the longest width before

Commit:
[fix/index-bar] Make Index Bar available in general file list (#469)

* #413 show index bar in general file list
- Index bar is only visible, if sorting is "Sort by name"
- Index bar is only visible, if more than one different letters are available
- Index bar is only visible, if more items are available as visible

* - adopt to new sort methods
- reverse index list, if sort direction is descendant

Commit:
Version Bump to 134

felix-schwarz committed:
Merge remote-tracking branch 'origin/master' into milestone/1.1

Commit:
Version Bump to 135

Commit:
Version Bump to 136

mneuwert committed:
Change behaviour of default expiration date (#476)

* Enabling expiration date switch if default expiration date is configured

Checking files_sharing.public.expire_date.expire_date.days server capability. If it is set then it is used as default date and the switch is enabled even if the expiration date is not enforced.

* Fixed QA issue (1)

* Fixed QA finding (2): limiting the date range if expiration date is enforced

mneuwert committed:
Permission increasing UX in reshares (#467)

* Not showing permissions which can’t be elevated

Affecting items shared with the current users which are being reshared

* Fixed an issue with wrong permissions displayed when creating public link

- Checking if permissions can be elevated for the public link
- Only viable options are displayed

* Fixed a trailing whitespace warning

* Fixed another compiler warning

* Inhereting permissions from the share when creating public link

* Fixed QA finding (4)

* Fixed QA finding (1)

* Fixed QA finding (2)

Commit:
Version Bump to 137

felix-schwarz committed:
[feature/itempolicy] Item Policy / Available Offline support (#456)

* Available Offline Support
- new MakeAvailableOfflineAction and MakeUnavailableOfflineAction actions
- new Available Offline Library section
	- easy access to all files available offline via Available Offline
	- new ItemPolicyTableViewController and ItemPolicyCell prepared for reuse for presenting other item policies
- Design: added SVG/TVG/PNG icons and cloud symbols for make available offline / make unavailable offline

ClientItemCell changes
- new ClientItemResolvingCell can resolve item from LocalID or path, new superclass of ShareClientItemCell and ItemPolicyCell
- string to use as content for title and detail are now provided via titleLabelString(for:) and detailLabelString(for:), updated via updateLabels(with:), making customization easier
- support for new available offline badge

ProgressSummarizer changes
- changed ProgressSummary from a struct to a class, addressing issue #451 ("Waiting for server response..." message sometimes not disappearing)

* - Adopt OCClaim to ensure files aren't deleted while they are being used in
	- DisplayViewController
	- DownloadItemsHUDViewController
	- FileListTableViewController
	- File Provider
- Add new Storage Settings to allow control over how long unused local copies are kept around
- DisplayViewController now uses OCClaims to ensure files aren't deleted while they are being viewed
- ClientItemCell uses new SDK APIs to
	- show "available offline" badges for folders
	- to indicate if an item has been made available offline directly (solid icon) or indirectly (dimmed icon)
	- keep the cloud status icon updated following changes
- Change "+" icon to "•••" icon in file list, rename plusButton to folderAction everywhere
- Action.provideAlertAction() now applies padding to images if necessary so their text aligns at the same horizontal offset
- Fix typo in BookmarkInfoViewController and change wording from "Delete Offline Copies" to "Delete Local Copies" for consistency
- Update swiftlint.yml to silence ownCloudScreenshotsTests warnings

* - Update ios-sdk

* - Make changes requested in code review, adding missing localizable strings and removing extraneous code

* - Update SDK

* - Add downloadTriggerIdentifier and availableOfflinePolicyCoverage as properties that trigger the update of an item's visual representation (addressing finding 6 in #456)
- Update make available offline action to use new convertExistingLocalDownloads option (addressing finding 2 in #456)
- Update SDK (addressing finding 5 in #456)

* Items in moved Available Offline folders should no longer be cleared locally and re-downloaded, following under the hood improvements in the ios-sdk

* - Add initial support for cookies via SDK update

* - Temporarily disable some UI tests broken due to lack of database running

* - Fix issue (11) in #456 by updating the lastUsed date of already downloaded items when they are viewed

* - Added slider type to StaticTableViewRow
- changed storage settings to allow picking a timeframe via slider rather than a list of time frames

* - Change text from "Remove" to "Make unavailable offline" in Quick Access > Available Offline list (addressing issue (9))

* - Update BookmarkInfoViewController to provide users with the choice to also remove local copies of items marked as Available Offline, issuing a warning and giving the users a way to cancel, addressing issue (12) in #456.

* - Update ios-sdk
- Take advantage of the new key-value store and OCCoreSkipAvailableOfflineKey to make sure available offline files are not immediately downloaded again after clearing the vault from local copies of available offline files.
- Logging into an account removes any previously set OCCoreSkipAvailableOfflineKey from the key value store
- Move Bookmark locking to an OCBookmarkManager category (next stop: use the new key-value store and bake support for locking bookmarks directly into the SDK)

* - Add missing localization

* Update SDK with latest fixes

* - Fix SettingsTests bugs

* - Apply additional fixes to UI tests and disable the remaining failing ones

* - Wait for connection initialization to complete before requesting sharing info
- Use latest SDK with cookie filtering

* - Turn off Request/Response log tags only filter in project file

* - Disable cookie support

* - Address (13) via SDK update

* link against newest development SDK

* Fixes for UI testing:
- use new SDK version that returns errors if the SQLite DB has not been opened
- MockOCCore.state now returns .running to avoid queueing of PROPFINDs

* - Fix unit tests

* - UI testing: make MockOCCore.state no longer return .running due to side effects
- fixed conflicting constraints in two places

* - force-delete bookmarks in UI tests if timeout has passed, log error but don't assert in that case
- handle case where a swipe on table cells in DeleteBookmarkTests triggers deletion outright

* - Further hardening of SettingsTests against timing differences local vs. CI

* - make SettingsTest.testCheckMoreItems() more robust

Commit:
Version Bump to 138

Commit:
[feature/reorder buttons] Reorder Navigation Bar Actions (#478)

* Available Offline Support
- new MakeAvailableOfflineAction and MakeUnavailableOfflineAction actions
- new Available Offline Library section
	- easy access to all files available offline via Available Offline
	- new ItemPolicyTableViewController and ItemPolicyCell prepared for reuse for presenting other item policies
- Design: added SVG/TVG/PNG icons and cloud symbols for make available offline / make unavailable offline

ClientItemCell changes
- new ClientItemResolvingCell can resolve item from LocalID or path, new superclass of ShareClientItemCell and ItemPolicyCell
- string to use as content for title and detail are now provided via titleLabelString(for:) and detailLabelString(for:), updated via updateLabels(with:), making customization easier
- support for new available offline badge

ProgressSummarizer changes
- changed ProgressSummary from a struct to a class, addressing issue #451 ("Waiting for server response..." message sometimes not disappearing)

* - Adopt OCClaim to ensure files aren't deleted while they are being used in
	- DisplayViewController
	- DownloadItemsHUDViewController
	- FileListTableViewController
	- File Provider
- Add new Storage Settings to allow control over how long unused local copies are kept around
- DisplayViewController now uses OCClaims to ensure files aren't deleted while they are being viewed
- ClientItemCell uses new SDK APIs to
	- show "available offline" badges for folders
	- to indicate if an item has been made available offline directly (solid icon) or indirectly (dimmed icon)
	- keep the cloud status icon updated following changes
- Change "+" icon to "•••" icon in file list, rename plusButton to folderAction everywhere
- Action.provideAlertAction() now applies padding to images if necessary so their text aligns at the same horizontal offset
- Fix typo in BookmarkInfoViewController and change wording from "Delete Offline Copies" to "Delete Local Copies" for consistency
- Update swiftlint.yml to silence ownCloudScreenshotsTests warnings

* - Update ios-sdk

* - Make changes requested in code review, adding missing localizable strings and removing extraneous code

* #477 reorganize navigation bar with create and more button and moved select button to sort menu

* added safeAreaLayoutGuide.rightAnchor

* - Update SDK

* changed "Select" image

* - Add downloadTriggerIdentifier and availableOfflinePolicyCoverage as properties that trigger the update of an item's visual representation (addressing finding 6 in #456)
- Update make available offline action to use new convertExistingLocalDownloads option (addressing finding 2 in #456)
- Update SDK (addressing finding 5 in #456)

* Items in moved Available Offline folders should no longer be cleared locally and re-downloaded, following under the hood improvements in the ios-sdk

* - Temporarily disable some UI tests broken due to lack of database running

* - Fix issue (11) in #456 by updating the lastUsed date of already downloaded items when they are viewed

* - Added slider type to StaticTableViewRow
- changed storage settings to allow picking a timeframe via slider rather than a list of time frames

* - Change text from "Remove" to "Make unavailable offline" in Quick Access > Available Offline list (addressing issue (9))

* - Update BookmarkInfoViewController to provide users with the choice to also remove local copies of items marked as Available Offline, issuing a warning and giving the users a way to cancel, addressing issue (12) in #456.

* addresses:
- missing multiselection UI in toolbar
- missing select all / deselect all button in navigation bar

* set if selectButton should be shown

* disable temporary some tests due to bitrise failures

Commit:
Version Bump to 139

mneuwert committed:
Feature/task scheduling (#484)

* #386 Added relevant classes

* First draft of the scheduled task manager

* Just changed some comments

* Implemented background fetch

* Implemented background update fetch

* Clean up of the code

* Fixed merge problems from previous commit

And eliminated some warnings

* Small changes in background fetch operation

* Some refactoring done

* Added some user defaults for media upload

* Available Offline Support
- new MakeAvailableOfflineAction and MakeUnavailableOfflineAction actions
- new Available Offline Library section
	- easy access to all files available offline via Available Offline
	- new ItemPolicyTableViewController and ItemPolicyCell prepared for reuse for presenting other item policies
- Design: added SVG/TVG/PNG icons and cloud symbols for make available offline / make unavailable offline

ClientItemCell changes
- new ClientItemResolvingCell can resolve item from LocalID or path, new superclass of ShareClientItemCell and ItemPolicyCell
- string to use as content for title and detail are now provided via titleLabelString(for:) and detailLabelString(for:), updated via updateLabels(with:), making customization easier
- support for new available offline badge

ProgressSummarizer changes
- changed ProgressSummary from a struct to a class, addressing issue #451 ("Waiting for server response..." message sometimes not disappearing)

* UI for bookmark / upload path selection

* Added dummy implementation of media upload task

* - Adopt OCClaim to ensure files aren't deleted while they are being used in
	- DisplayViewController
	- DownloadItemsHUDViewController
	- FileListTableViewController
	- File Provider
- Add new Storage Settings to allow control over how long unused local copies are kept around
- DisplayViewController now uses OCClaims to ensure files aren't deleted while they are being viewed
- ClientItemCell uses new SDK APIs to
	- show "available offline" badges for folders
	- to indicate if an item has been made available offline directly (solid icon) or indirectly (dimmed icon)
	- keep the cloud status icon updated following changes
- Change "+" icon to "•••" icon in file list, rename plusButton to folderAction everywhere
- Action.provideAlertAction() now applies padding to images if necessary so their text aligns at the same horizontal offset
- Fix typo in BookmarkInfoViewController and change wording from "Delete Offline Copies" to "Delete Local Copies" for consistency
- Update swiftlint.yml to silence ownCloudScreenshotsTests warnings

* - Update ios-sdk

* - Make changes requested in code review, adding missing localizable strings and removing extraneous code

* - Update SDK

* Refactored current implementation of media upload

* Added method to fetch photos from photo library

* - Add downloadTriggerIdentifier and availableOfflinePolicyCoverage as properties that trigger the update of an item's visual representation (addressing finding 6 in #456)
- Update make available offline action to use new convertExistingLocalDownloads option (addressing finding 2 in #456)
- Update SDK (addressing finding 5 in #456)

* Items in moved Available Offline folders should no longer be cleared locally and re-downloaded, following under the hood improvements in the ios-sdk

* - Add initial support for cookies via SDK update

* First draft of instant photo upload implementation

* Calling task completion handler after upload was finished

* - Temporarily disable some UI tests broken due to lack of database running

* - Fix issue (11) in #456 by updating the lastUsed date of already downloaded items when they are viewed

* - Added slider type to StaticTableViewRow
- changed storage settings to allow picking a timeframe via slider rather than a list of time frames

* - Change text from "Remove" to "Make unavailable offline" in Quick Access > Available Offline list (addressing issue (9))

* - Update BookmarkInfoViewController to provide users with the choice to also remove local copies of items marked as Available Offline, issuing a warning and giving the users a way to cancel, addressing issue (12) in #456.

* Fixed QA finding (1)

* Using PHAsset.creationDate to check which media files shall be instantly uploaded

* Fixed naming of class / group

* Moved Tasks folder

* Fixed some review findings

* - Fix issue where a busy bookmark alert is followed by duplicating the bookmark

* Fixed some review findings

* Improved background update fetching

* Fixed some issues in media upload

* Fixed a type in settings key

* Improvements in the instant media upload task

* Added missing returnCore in the selection of the instant upload path

* Fixed QA finding (3)

* Fixes for the correct core returning / requesting

* Small cosmetic code changes

* Fixed video and serialized photo / video uploads

* Moved upload() method to UploadBaseAction class

* Some refactoring in InstantMediaUploadTaskExtension

* Improvements for bookmark / path selection for instant upload

* Remove instant upload configuration if bookmark is not available

* minor refactoring

* Added possibility to cancel account selection

* Small fix: reset the path if new account is selected

* Using item tracking instead of OCQuery

- Disabling instant photo upload if target directory is removed
- Item tracking is potentially faster since it can use cached item information and is not always issuing PROPFIND

* Made some ivars private

* Detect in settings that instant upload path is gone

In this case instant upload is disabled

* - Update ios-sdk
- Take advantage of the new key-value store and OCCoreSkipAvailableOfflineKey to make sure available offline files are not immediately downloaded again after clearing the vault from local copies of available offline files.
- Logging into an account removes any previously set OCCoreSkipAvailableOfflineKey from the key value store
- Move Bookmark locking to an OCBookmarkManager category (next stop: use the new key-value store and bake support for locking bookmarks directly into the SDK)

* - Add missing localization

* Update SDK with latest fixes

* - Fix SettingsTests bugs

* - Apply additional fixes to UI tests and disable the remaining failing ones

* - Wait for connection initialization to complete before requesting sharing info
- Use latest SDK with cookie filtering

* - Turn off Request/Response log tags only filter in project file

* - Disable cookie support

* - Address (13) via SDK update

* link against newest development SDK

* Showing warning that instant upload was disabled

* Fixes for UI testing:
- use new SDK version that returns errors if the SQLite DB has not been opened
- MockOCCore.state now returns .running to avoid queueing of PROPFINDs

* - Fix unit tests

* Added alert shown when in settings when instant upload is disabled

* Added fetchUpdates() within which target folder tracking is started

* - UI testing: make MockOCCore.state no longer return .running due to side effects
- fixed conflicting constraints in two places

* - force-delete bookmarks in UI tests if timeout has passed, log error but don't assert in that case
- handle case where a swipe on table cells in DeleteBookmarkTests triggers deletion outright

* - Further hardening of SettingsTests against timing differences local vs. CI

* - make SettingsTest.testCheckMoreItems() more robust

* Update to the latest changes in the SDK

Those changes reduce memory consumption used for hash calculation

* Change required to accomodate latest SDK changes

* Less parallel photo uploads

* - update SDK to address stuck sync action issues
- no longer auto-open the last selected bookmark when launched by iOS in the background. Instead, the last selected bookmark is now auto-openend only when the app comes to the foreground.
- update FileProviderInterfaceManager to better support the case where the app ships without File Provider.

* Fixed compilation warning

* - Add ATS exemption for File Provider

* - Update SDK

* - Update SDK + add colored debugging option to scheme

* Addressed code review comments

* Removed unneeded weak self

* Removed another unwanted weak self

* - Fix OCEvent-dropping issue via SDK update with fix
- Make UploadMediaAction use OCBackgroundTask to protect while exporting

* - Added missing OCBackgroundTask.start call in UploadMediaAction
- Update SDK

* Fixed missing copyright information and some warnings

Commit:
Version Bump to 140

mneuwert committed:
Media upload pending flag added

- Displaying a warning alert on the next launch, in case media upload couldn’t be completed due e.g. app being terminated in a backround or even due to an app crash.
- Fixed an issue with DispatchGroup.notify() called prematurely in MediaUploadQueue.uploadAssets()

hosy committed:
- sortbar was sometimes hidden
- fixing layout issue

hosy committed:
fixed location identifier for actions

mneuwert committed:
Added more logs in PHAsset+Upload extension

mneuwert committed:
Fixes (1) and (2)

Commit:
disable beta warning

mneuwert committed:
Fixes issue (3) with downloads being cancelled by preview

mneuwert committed:
Merge branch 'release/1.1.0' of https://github.com/owncloud/ios-app into release/1.1.0

felix-schwarz committed:
- Update SDK to address finding (4) in #461

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/owncloud/ios-app?pullRequest=499) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/owncloud/ios-app?pullRequest=499) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: jesmrec<br/>:white_check_mark: mneuwert<br/>:x: felix-schwarz<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/owncloud/ios-app?pullRequest=499) it.</sub>

felix-schwarz committed:
- Update SDK to solve remaining of finding (4)

felix-schwarz committed:
- Update ios-sdk to include latest changes
- Fix SwiftLint warning in DisplayHostViewController.swift

hosy committed:
updated changelog for version 1.1.0

felix-schwarz committed:
- change presentation of "Make unavailable offline" action to filled icon + "Available offline" + checkmark
- optimize PNG images to save space
- fix lack of OAuth2 renewed token propagation (#501: token renewal from Files App) via ios-sdk update

hosy committed:
added missing localization string

felix-schwarz committed:
- Update ios-sdk to satisfy new requirements for using ASAuthorization in iOS 13

felix-schwarz committed:
- Fix finding (11) in #461 via SDK update

mneuwert committed:
Fixed issue (6)

felix-schwarz committed:
- Add debug output to verify theory on stuck uploads issue

felix-schwarz committed:
- Update SDK to add improved logging and allowing to enable use of OCCores after they were returned and stopped ("zombie core detection")

felix-schwarz committed:
- Update SDK
- Fix retain cycle in MediaUploadSettingsSection that could also lead to a zombie core

hosy committed:
using the newest development SDK:
- solving showing certificate error in document browser (file provider)
- solved importFile on main thread issue

hosy committed:
Version Bump to 141

hosy committed:
- added new var to get beta warning state
- disabled beta warning, beta build
- using the newest development SDK

hosy committed:
set new build number

hosy committed:
using master SDK version 1.1.0

Commit:
Translation sync (#458)

* [tx] updated from transifex

* Removed the es lang because was wrong and it is not necessary to map it (es -> es) (#310)

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* use newest SDK version

* changed file encoding for localization files

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* - using correct SDK
- solved encoding problems for localization files

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* change version number 1.0 -> 1.0.1

* Fixed QA finding (1). "Share item" menu item was not visible, if item was not shared

* Fix finding (3)

* Fixed an issue with go-to button not being shown in PDF viewer (#403)

* Fixed an issue with go-to button not being shown in PDF viewer

* Added accessibility labels to bar button items in the PDF viewer

* Version Bump to 121

* - Action class:
	- extend ActionWillRunHandler with a completionHandler and adding a new .perform() method to wrap .willRun() and .run(), allowing for actions to trigger view controller dismissals, wait for them to finish and then present their own view controllers
- Card upgrades, improvement and fixes
	- fix MoreViewController "shaking" on initial presentation
	- create new CardPresentationSizing protocol to allow view controllers to provide sizing information to CardPresentationController
	- allow disabling the handle and the ability to dismiss by tapping in the background
	- create CardViewController as base class for future view controllers that should be presented as cards
 - Add new DownloadItemsHUDViewController class (a CardViewController subclass) to modally download files, provide a summary, allow cancellation and simplified retrieval of the corresponding OCFile objects
- Clean up OpenInAction and rewrite most of it to use DownloadItemsHUDViewController
- KVOWaiter allows waiting for an object's value to change, check if the change meets the criteria and then run a block once. Ended up not being used and is therefore not included in the builds. But since it may be useful in the future, left the source code regardless.
- Add ProgressSummary.update(progressView:) convenience method

* [tx] updated from transifex

* - Switch to release/1.0.1 branch for ios-sdk

* - Add "Preparing" message to DownloadItemsHUDViewController while scheduling downloads
- Update ios-sdk, fixing findings (5) + (6)

* - Fix premature release of UIDocumentInteractionController in OpenInAction

* using correct release SDK

* [tx] updated from transifex

* - Add support for Progress.isCancelled to ProgressSummarizer

* Version Bump to 122

* - Addressing (5) via SDK update rescheduling/retrying dropped requests

* - Addressing (5) via SDK update rescheduling/retrying dropped requests

* - Make File Provider use new SDK option to return immediately from a thumbnail request when offline

* [tx] updated from transifex

* Version Bump to 123

* - Add new APP_SHORT_VERSION variable to build settings
- Make Info.plist for app, File Provider and File Provider UI use it

* - Fix remaining (5) issue for cases where the upload succeeded but the server response was cut off, prompting a retry and resulting in a "item already exists" error

* - Update ios-sdk

* [tx] updated from transifex

* [tx] updated from transifex

* changed wrong file encoding from UTF-16LE to UTF-8

* QA finding (14) removed ownCloud name from localizable stings

* removed ownCloud string in localizable strings

* fixed merge conclicts

* using newest release SDK

* changed file encoding from UTF-16LE to UTF-8

* changed file encoding from UTF-16LE to UTF-8

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* [tx] updated from transifex

* using correct SDK

* removed unused string files for fastlane screenshot creation

* fixed strange text encoding (Transifex error)=

* [tx] updated from transifex

* - new localization values from Transifex
- changed wrong/too long localization strings
- added missing localizations

* added new localization language Basque

* added localization files for fastlane screenshot generator

* updated to newest SDK after localization changes

hosy commented on ownCloud File Provider/Info.plist:
```diff
@@ -39,5 +39,10 @@
 	<string>group.com.owncloud.ios-app</string>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
```
I think it is needed for insecure http connections and it was missing for this target
